frontend/src/pages/PersonalPage.jsx

```diff
@@ -176,33 +170,32 @@ function PersonalPage(props) {
                     <div className="d-flex flex-column left-side">
                         <ModalHeader title={`Hi ${user}`} />
                         <div className="d-flex flex-column align-items-left">
-                            <ModalLeftWindow
+                            <LeftPanel
                                 body={
-                                    <div className="d-flex flex row">
+                                    <div className="flex-row">
                                         <ModalHeader title="My Friends" />
                                         <div className="d-flex row agenda">
                                             {friends.map((friend) => (
                                                 <h3>{friend.username}</h3>
                                             ))}
                                         </div>
-                                        <div className="d-flex">
+                                        <div className="flex-row">
                                             <Button
                                                 title="ADD FRIEND"
```

Comment:
I think these panel buttons are a bit large, maybe we can add a small property to the Button component, which conditionally adds the ``small`` class to the button, like this: 
![image](https://user-images.githubusercontent.com/58454072/141787298-788c07cd-c57e-440d-9ed9-10834c6940c4.png)

Reply:
Thanks! Changed to small.